### ISSUE TITLE
xrdp-0.9.3.tar.gz contains unnecessary *.service files generated from *.service.in

### ISSUE THREAD

**Issue by ftake** [neutrinolabs/xrdp#848]
The tar.gz archive available on the Release page contains unnecessary xrdp.service and xrdp-sessman.service. These files prevent them from regenerated during make according to the real system directories, in other words, *.service always point at /usr/local/sbin.

**Comment by metalefty** [comment]
Running `make clean` after `./configure` is a workaround.

**Comment by metalefty** [comment]
@ftake can you test #851?

Clone my repository and make a tarball.

```
./bootstrap && ./configure && make distcheck
```

Then, make sure *.service files are not included in the distribution.

Finally, make && make install and confirm *.service files are installed and they refer correct path specified by `--prefix` flag.

```
./configure --prefix=/some/where && make && make install DESTDIR=/tmp/somewhere
```

If it seems OK, I'm going to make a new release 0.9.3.1.

**Comment by ftake** [comment]
> ./bootstrap && ./configure && make distcheck

confirmed the generated archive does not contain *.service.

> ./configure --prefix=/some/where && make && make install DESTDIR=/tmp/somewhere

Giving --prefix=/usr, confirmed the path become /usr/sbin too.

Thanks!

**Comment by metalefty** [comment]
I've released v0.9.3.1.
